Orange/widgets/gui.py

```diff
@@ -552,6 +552,14 @@ def widgetBox(widget, box=None, orientation='vertical', margin=None, spacing=4,
     return b
 
 
+def horizontalBox(*args, **kwargs):
```

Comment:
This branch was not based on master but on the one which I pushed into PR #1014. It thus contained the same commit that's already there (it no longer does, since I amended that commit).

I thought that merging this PR into master would skip the commit that's already in the master. Would it? Is this an appropriate way of basing your work on your other branches that were not merged yet?